src/components/metadata/metadata.js

```diff
@@ -203,7 +201,10 @@ const MetaData = ({
                   />
                 )}
                 <MetaDataRow label="Run Command:" visible={Boolean(runCommand)}>
-                  <CommandCopier command={runCommand} />
+                  <CommandCopier
+                    command={runCommand}
+                    isCommand={metadata?.runCommand}
```

Comment:
Is there a difference between `runCommand` and `metadata?.runCommand`?

I ask because I wonder if you can do `Boolean(runCommand)` for your new `isCommand` prop, the way it's done for the `visible` prop above?

Reply:
yeah, if you go top of the file. u will see we if there's no metadata?.runCommand we specify the runCommand to be that test.